ts_scripts/marsgen.py

```diff
@@ -0,0 +1,116 @@
+import argparse
+import json
+import os
+import sys
+import urllib.request
+
+REPO_ROOT = os.path.join(os.path.dirname(os.path.abspath(__file__)), "..")
+sys.path.append(REPO_ROOT)
+model_store_dir = os.path.join(REPO_ROOT, "model_store")
+os.makedirs(model_store_dir, exist_ok=True)
+
+def generate_mars(mar_config):
+
+    with open(mar_config) as f:
+        models = json.loads(f.read())
+
+        for model in models:
```

Comment:
Nit: Once `models` has been read from `mar_config`, this particular 'for' block need not be within the context of file 'open'. The whole indentation can move back once.